Xamarin.Forms.Platform.Android/Renderers/DatePickerRenderer.cs

```diff
@@ -175,7 +176,18 @@ void OnCancelButtonClicked(object sender, EventArgs e)
 
 		void SetDate(DateTime date)
 		{
-			EditText.Text = date.ToString(Element.Format);
+			if (Element.Format == "")
+			{
+				EditText.Text = date.ToShortDateString();
+			}
+			else if (Element.Format.Contains('/'))
```

Comment:
Can `Element.Format` ever be null? If so, this will cause a `NullReferenceException`. You can prevent that by using a [null-conditional operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-) here, and you probably want to review other places where you call a method from `Element.Format` to make sure you're accounting for a null value.

Reply:
When I switched Element.Format to null, it didn't throw a NullReferenceException. However, I did switch the first if to IsNullOrWhitespace.

Reply:
I switched the first check to IsNullOrWhitespace, and when I tested setting the Element.Format to null, it didn't cause a NullReferenceException.